test/evaluation/limitDisclosureEvaluationHandler.spec.ts

```
@@ -2,34 +2,117 @@ import fs from 'fs';
+  it("should return ok if verifiable Credential doesn't have the etc field", () => {
     const pdSchema: PresentationDefinition = getFile(
       './test/dif_pe_examples/pd/pd-simple-schema-age-predicate.json'
     ).presentation_definition;
     const vpSimple: VerifiablePresentation = getFile('./test/dif_pe_examples/vp/vp-simple-age-predicate.json');
     const evaluationClient: EvaluationClient = new EvaluationClient();
-    evaluationClient.evaluate(pdSchema, vpSimple.verifiableCredential, HOLDER_DID);
-    expect(evaluationClient.verifiableCredential[0].credentialSubject['etc']).toEqual(undefined);
+    evaluationClient.evaluate(pdSchema, vpSimple.verifiableCredential, HOLDER_DID, LIMIT_DISCLOSURE_SIGNATURE_SUITES);
+    expect(evaluationClient.verifiableCredential[0].credentialSubject['etc']).toBeUndefined();
   });
 
-  it("should return ok if verifiable Credential doesn't have the birthPlace field", function () {
+  it("should return ok if verifiable Credential doesn't have the birthPlace field", () => {
     const pdSchema: PresentationDefinition = getFile(
       './test/dif_pe_examples/pd/pd-schema-multiple-constraints.json'
     ).presentation_definition;
     const vpSimple: VerifiablePresentation = getFile('./test/dif_pe_examples/vp/vp-multiple-constraints.json');
     pdSchema.input_descriptors[0].schema.push({ uri: 'https://www.w3.org/2018/credentials/v1' });
     const evaluationClient: EvaluationClient = new EvaluationClient();
-    evaluationClient.evaluate(pdSchema, vpSimple.verifiableCredential, HOLDER_DID);
-    expect(evaluationClient.verifiableCredential[0]['birthPlace']).toEqual(undefined);
+    evaluationClient.evaluate(pdSchema, vpSimple.verifiableCredential, HOLDER_DID, LIMIT_DISCLOSURE_SIGNATURE_SUITES);
+    expect(evaluationClient.verifiableCredential[0].credentialSubject['birthPlace']).toBeUndefined();
+  });
+
+  it('limit disclosure should not be supported', () => {
```

if limit disclousure is not supported report an error

this seems to be a better title

---

Actually I found out that it() is meant to be part of the title, so what about 'should report an error if limit disclosure is not supported', so you read it 'it should report an error if limit disclosure is not supported'?

---

Yes, that is how it is supposed to be interpretted. your option looks good as well.